### ISSUE TITLE
Missing labels from <input> elements

### ISSUE THREAD

**Issue by bobbysebolao** [FACN7/week2-akn-api#27]
![Screen Shot 2019-09-19 at 14 44 23](https://user-images.githubusercontent.com/30030755/65241383-0c98fa80-daec-11e9-85c3-7fe0949807af.png)

Google's Lighthouse accessibility audit tool says you have an input element that doesn't have a label. Label text is important for visually impaired people who use screen readers (the screen readers read it out loud to them when they focus on the input element)
